app/views/pages/privacy.html.erb

```diff
@@ -1,52 +1,71 @@
-
-      <h2 class="govuk-heading-l">Why our use of your personal data is lawful</h2>
-      <p class="govuk-body">In order for our use of your personal data to be lawful, we need to meet one (or more) conditions in the data protection legislation. For the purpose of this project, this processing is necessary to complete a task in the public interest.</p>
-
-      <h2 class="govuk-heading-l">How long we will keep your personal data</h2>
-      <p class="govuk-body">We will only keep your personal data for as long as we need it for the purpose(s) of this piece of work, after which point it will be securely destroyed. Please note that under data protection legislation and in compliance with the relevant data processing conditions, we can lawfully keep personal data processed purely for research and statistical purposes indefinitely.</p>
-
-      <h2 class="govuk-heading-l">Your data protection rights</h2>
-      <p class="govuk-body">You have the right:</p>
-      <ul class="govuk-list govuk-list--bullet">
-        <li>to ask us for access to information about you that we hold</li>
-        <li>to have your personal data rectified, if it is inaccurate or incomplete</li>
-        <li>to request the deletion or removal of personal data where there is no compelling reason for its continued processing</li>
-        <li>to restrict our processing of your personal data (i.e. permitting its storage but no further processing)</li>
-        <li>to object to direct marketing (including profiling) and processing for the purposes of scientific/historical research and statistics</li>
-        <li>not to be subject to decisions based purely on automated processing where it produces a legal or similarly significant effect on you</li>
-      </ul>
-
-      <p class="govuk-body">
-        You can find more information about how we handle personal data in our
-        <%= govuk_link_to('personal information charter', 'https://www.gov.uk/government/organisations/department-for-education/about/personal-information-charter') %>.
-      </p>
-
-      <h2 class="govuk-heading-l">The right to lodge a complaint</h2>
-      <p class="govuk-body">
-        You have the right to raise any concerns with the Information Commissioner’s Office (ICO) via their website at
-        <%= govuk_link_to('https://ico.org.uk/make-a-complaint/', 'https://ico.org.uk/make-a-complaint/') %>.
-      </p>
-
-      <h2 class="govuk-heading-l">Last updated</h2>
-      <p class="govuk-body">This version was last updated on 30 April 2018.</p>
-
-      <h2 class="govuk-heading-l">Contact us</h2>
-      <p class="govuk-body">Email <%= bat_contact_mail_to %> if you have any questions about how your personal information will be processed.</p>
-    </div>
+<% content_for :page_title, "Privacy notice" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Privacy notice</h1>
```

Comment:
Did we mean to change this to `notice` or the full heading as per the doc? Also should the link still be `privacy policy` in the footer etc? @Kateharries

Reply:
Clarified with Victoria offline